interfaces/smtp/v0/README.md

```diff
@@ -0,0 +1,64 @@
+# `smtp`
+
+## Overview
+
+This relation interface specification describes the expected behavior of charms integrating over the `smtp` Juju interface to provide or consume SMTP data.
+
+## Usage
+
+In most cases, this will be accomplished using the [smtp library](https://github.com/canonical/smtp-integrator-operator/blob/main/lib/charms/smtp_integrator/v0/smtp.py), although charm developers are free to provide alternative libraries as long as they fulfil the behavioural and schematic requirements described in this document.
+
+## Direction
+
+The `smtp` interface implements a provider/requirer pattern.
+The requirer is a charm that requires SMTP details to connect to an SMTP server, and the provider is a charm holding those details.
+
+```mermaid
+flowchart TD
+    Provider -- host, port, user, password_id, auth_type, transport_security, domain --> Requirer
+```
+
+or alternatively, if secrets are not supported by both sides
+
+```mermaid
+flowchart TD
+    Provider -- host, port, user, password, auth_type, transport_security, domain --> Requirer
+```
+
+## Behavior
+
+The requirer and the provider must adhere to a certain set of criteria to be considered compatible with the interface.
+
+### Provider
+
+- Is expected to publish the `host` in the relation databag.
+- Is expected to publish the `port` in the relation databag.
+- Is expected to publish the `auth_type` in the relation databag.
+- Is expected to publish the `transport_security` in the relation databag.
```

Comment:
These expectations aren't really useful as they're all implicitly provided through the schema. This section is supposed to describe the behavior, rather than the schema itself. See https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/grafana_auth/v0/README.md#behavior for an example.

Reply:
Noted. Thx